CustomCraftSML/Serialization/Entries/AddedRecipe.cs

```diff
@@ -92,12 +91,26 @@ public override bool SendToSMLHelper()
             }
             catch (Exception ex)
             {
-                QuickLogger.Error($"Exception thrown while handling Added Recipe '{this.ItemID}'{Environment.NewLine}{ex}");
+                QuickLogger.Error($"Exception thrown while handling Added Recipe '{this.ItemID}' from  from {this.Origin}", ex);
                 return false;
             }
         }
 
         protected void HandleAddedRecipe(short defaultCraftAmount = 1)
+        {
+            TechData replacement = CreateRecipeTechData(defaultCraftAmount);
+
+            CraftDataHandler.SetTechData(this.TechType, replacement);
+            QuickLogger.Message($"Adding new recipe for '{this.ItemID}'");
```

Comment:
`QuickLogger` needs a logging level configuration.  
All `Message` level goes should be configured as `Debug` logs now.  
Allow the user to enable `Debug` logs but have them turned off by default, with only `Warning` and `Error` logs showing.

A new `Info` log level should be added to include important summary data only.